sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/ExtractBenchmark.scala

```diff
@@ -39,7 +39,9 @@ object ExtractBenchmark extends SqlBasedBenchmark {
 
   private def doBenchmark(cardinality: Long, exprs: String*): Unit = {
     val sinceSecond = Instant.parse("2010-01-01T00:00:00Z").getEpochSecond
-    withSQLConf(SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> "true") {
+    withSQLConf(
+      SQLConf.LEGACY_INTERVAL_ENABLED.key -> "true",
+      SQLConf.WHOLESTAGE_CODEGEN_ENABLED.key -> "true") {
```

Comment:
BTW, I asked (offline) to don't generate the benchmark results because I plan to regenerate everything after https://github.com/apache/spark/pull/32015